src/app/infrastucture/format-media-type/format-media-type.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { MovieCollection, SeriesCollection } from 'src/app/moviedb-collections/moviedb-collection';
+import { CardContents } from '../ui-card/ui-card';
+
+@Pipe({
+  name: 'formatMediaType',
+})
+export class FormatMediaTypePipe implements PipeTransform {
+  transform(value: SeriesCollection | MovieCollection): CardContents {
+    if ('title' in value) {
```

Comment:
It's better to use the "mediaType" property here. Because "name" is not a defining property.

What I mean is that if for some reason "name" is renamed to "name", we will get strange behavior.